pyproject.toml

```diff
@@ -167,3 +194,6 @@ ignore = [
 docstring-quotes = "double"
 multiline-quotes = "double"
 inline-quotes = "single"
+
+[tool.pytest.ini_options]
+asyncio_default_fixture_loop_scope = "session"
```

Comment:
its just because pytest complains if you dont, I reverted this for now in [6ad722a](https://github.com/MarshalX/atproto/pull/451/commits/6ad722a5964483d3cc00d601c59db7126990b658)